wallet-new/src/Cardano/Wallet/Client.hs

```diff
@@ -78,7 +88,7 @@ data WalletClient m
     -- transactions endpoints
     , postTransaction
          :: Payment -> Resp m Transaction
-    , getTransactionIndex
+    , getTransactionHistory
```

Comment:
Please revert the cosmetic changes to the API. This is going to change a bit anyway when the client is merged; I'd prefer not to have more conflicts than necessary.

Reply:
Done. I do think the naming is a bit off.